crates/starknet-types-core/src/hash/pedersen.rs

```diff
@@ -26,12 +29,32 @@ impl StarkHash for Pedersen {
             });
         Felt(PedersenStarkCurve::hash(&current_hash, &data_len.0))
     }
+
+    /// Computes the Pedersen hash of a single Felt
+    ///
+    /// Warning: there is room for collision as:
+    /// pedersen(value) and pedersen(value, 0) will return the same values
```

Comment:
/// Pedersen::hash_singlevalue) and Pedersen::hash(value, 0) will return the same values